recipes/tllist/all/conanfile.py

```diff
@@ -0,0 +1,33 @@
+from conans import ConanFile, tools
+
+
+required_conan_version = ">=1.36.0"
+
+
+class TllistConan(ConanFile):
+    name = "tllist"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://codeberg.org/dnkl/tllist"
+    description = "A C header file only implementation of a typed linked list."
+    topics = ("list", "utils", "typed-linked-list")
+    settings = "os", "arch", "build_type", "compiler"
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
```

Comment:
No idea why it fails on windows and I don't have the time to figure it out. It fails on the test_package side. As the original PR was restricting build different from Linux, I am ok restricting them now only for windows and adding a proper fix later.
```suggestion
    def validate(self):
        # FIXME: Fails in test_pacakge with the Windows configuration
        if self.settings.os == "Windows":
            raise ConanInvalidConfiguration("Windows not supported")

```